ledger/metrics/src/main/scala/com/daml/telemetry/Telemetry.scala

```diff
@@ -85,6 +91,11 @@ abstract class DefaultTelemetry(override protected val tracer: Tracer) extends T
     }
   }
 
+  override def contextFromOpenTelemetryContext(context: Context): TelemetryContext =
```

Comment:
so how am i supposed to call this?

Reply:
@danilofaria I would go with `DefaultTelemetry.contextFromOpenTelemetryContext(rawContext)`.